system/services/enum/EnumService.cfc

```diff
@@ -45,6 +45,62 @@ component {
 		return itemsWithLabelsAndDescriptions;
 	}
 
+	public struct function getStructItems( required string enum, string value = "" ) {
+		var enums                          = _getConfiguredEnums();
+		var rawItems                       = enums[ arguments.enum ] ?: [];
+		var value                          = arguments.value         ?: "";
+		var itemsWithLabelsAndDescriptions = {};
+		var valueEnumResult                = {};
+
+		for( var itemId in rawItems ) {
+			itemsWithLabelsAndDescriptions[ itemId ] = {
+				  label       = $translateResource( uri="enum.#arguments.enum#:#itemId#.label"      , defaultValue=itemId )
+				, description = $translateResource( uri="enum.#arguments.enum#:#itemId#.description", defaultValue=""     )
+			}
+		}
+
+		if ( !isEmpty( arguments.value ) ) {
+			for ( item in listToArray( value, "," ) ) {
```

Comment:
Missing `var` here for the `item` variable. But actually, this loop and condition on 62 could be removed (see comment above).